recipes/pcre/all/conanfile.py

```diff
@@ -91,3 +90,5 @@ def package_info(self):
             self.cpp_info.libs = ['pcreposix', 'pcre']
         if not self.options.shared:
             self.cpp_info.defines.append("PCRE_STATIC=1")
+        if conan_version >= tools.Version("1.21.0"):
```

Comment:
Do not include Conan client version conditionals in recipes. The build service will always run with latest Conan version. The Conan version will be updated soon.

Reply:
Actually, I have put the check for consumers, not for the build server. Or is it implicit that conan-center-index recipes always require the latest conan version?

Reply:
https://docs.conan.io/en/latest/introduction.html#stable

> This means that public repositories, like conan-center assume the use of the latest version of the Conan client, and using an older version may result in failure of packages and recipes created with a newer version of the client.

And it is implicit in https://github.com/conan-io/conan-center-index/wiki#before-start